qa/rpc-tests/mn_tickets.py

```diff
@@ -1865,6 +1866,88 @@ def takedown_ticket_tests(self):
         # ...
         print("Take down tickets tested")
 
+    # ===============================================================================================================
+    def username_ticket_tests(self):
+        print("== Username Tickets test ==")
+        # Register first time
+
+        try:
```

Comment:
we need try/catch to catch the error and assert if it is the expected error

Reply:
Yes, it is not needed if the test ran as expected - no error shows up.
But what if someone implementing something that affecting current tests? This try/catch will save us 1 re-run time(which is taking like 2 hours) to check what is the error.
Since this is just test cases file. I don't see the reason why we have to save try/catch here.

Reply:
when I see try/catch - I expect to get error here. In "mn_tickets" tests we are trying to separate working code and code with errors.
I thought exception will come any case if exist. But present try/catch for working code looks a bit different from other implementation in the mn_tickets.

Reply:
Yes, the exception will come any case if exist. But without try catch, it will stop the test and throw that error.
What I mean is I want to let it run without stopping. In the case I mentioned (implementation changes that affect the test results), multiple errors will print together, so we can check them as 1 run time only.

Reply:
I agree with you that this can save time during testing but after testing it should be removed. Consider that some bug was added to the code after some time - you will be not informed about this new bug with exception as try/catch will hide it! Do you check the log of every call of every test during running different 60 tests? Should another coder to know your tests/log text and if is correct/incorrect?

Reply:
makes sense, will fix